phpstan.neon

```diff
@@ -293,3 +293,8 @@ parameters:
             paths:
                  - packages/package-builder/src/Parameter/ParameterProvider.php # 26
                  - packages/symfony-static-dumper/src/HttpFoundation/ControllerContentResolver.php # 37
+
+        -
+            message: '#Use namespaceName on Class_ node#'
```

Comment:
exclusion can be in symplify-rules.neon, covered in https://github.com/symplify/symplify/pull/2356

Reply:
I see, thanks.

I think we should allow $node->name only on adding to e.g. $shortClassName, so the intent to use short name is explicit in the code.

Reply:
Do you mean if variable name named `$shortClassName` the check should be skipped?

Reply:
Yes, it's just an idea.

```php
$name = $node->name; // FQN or short?

$name = $node->namespacedName; // clearly FQN!

$shortClassName = $node->name; // clearly short!
```

Reply:
I will try